Chapters/1-7-Chapter7.md

```diff
@@ -0,0 +1,493 @@
+
+"However, the codex of the senate makes clear that any interruption of any kind during deliberation is illegal. For this reason, I find you guilty of interruption of the senate during its deliberation."
+
+The conviction was expected, but it still hurt.
+
+"The sentence shall be a period of three La'To in Ulukoma. Do you understa-"
+
+The door opened in a rush, and the magistrate paused to see who caused the disturbance. Two lanterns were in view.
+
+"Fajro? What are you doing here?" the magistrate asked.
+
+I was thinking the same thing, but I quickly changed my mind when I saw the other face.
+
+"Kipnu! You came!" I shouted.
+
+He rushed to me and reached for my face with his free hand. Fajro walked in his slower pace toward us and attempted to bow to the magistrate.
+
+"Oh there isn't any need of that," the magistrate said.
+
+"Pardon my old habits. They've taken a permanent residence with me." He responded.
+
+He reached the other side of Kipnu and looked closely at me, trying to telegraph something but I did not know what.
+
+"I hope we came in time to defend Liffa," he said.
+
+The magistrate responded, "Well, it's a little bit late, but I'll permit it if you have something to say worth adding."
+
+The magistrate turned to me and asked, "Liffa, do you approve of Fajro's testimony?"
+
+"Only if Kipnu is allowed to speak as well."
+
+"Yes, that would be fine."
+
+The magistrate took his paper and charcoal out again and turned to Fajro, motioning for him to speak.
+
+Fajro began, "Liffa was helping me walk. I intended to interrupt the senate myself, and unwittingly inspired hae to finish the job when I was stopped by guards."
+
+I was aghast. He was taking the blame? For what?
+
+The magistrate stopped writing and laid his forehead into his hand. "That's quite a thing to admit, Fajro, but Liffa was still the one who actually interrupted the deliberation."
+
+He replied, "I understand. My only hope for my testimony is that you give a lighter sentence to a maman who was acting under my orders."
+
+"Well, the codex of the senate has pretty clear guidelines on sentencing for interruption. I would be risking my post if I let hae free without punishment."
+
+Kipnu turned to the magistrate and pleaded to him, "Please sir. Liffa had every positive intention. Thi doesn't deserve to be shipped off."
+
+Fajro chimed in, "Perhaps an alternative arrangement can be made. One that is beneficial to both Liffa and all of Palinan."
+
+Kipnu and the magistrate turned to him to hear more.
+
+"If Liffa is inclined, thi could work with me in Holrag on our observational study of the obstruction."
+
+I replied immediately, "I am not inclined."
+
+"Hold on, Liffa, let me finish. You would be working with our scientists to send a spacecraft up and figure out what we're dealing with. It would take no more than a la'tova, then you can head back home to Heiko."
+
+I retorted, "What use would a farmer be to scientists?"
+
+"An ex-engineer, you mean."
```

Comment:
😱 I guess it made my previous comment about something Liffa thought seeming OoC makes more sense.

OoC : Out of character